Libraries/WebSocket/WebSocket.js

```diff
@@ -92,6 +92,10 @@ class WebSocket extends EventTarget(...WEBSOCKET_EVENTS) {
       protocols = null;
     }
 
+    if (!/^wss?:\/\//.test(url)) {
```

Comment:
Your PR description claims that the issue arises when an URL is used with a scheme other than ws/wss/http/https, and according to https://github.com/facebook/react-native/blob/master/Libraries/WebSocket/RCTSRWebSocket.m#L318, `ws`, `wss`, `http`, `https` are all valid schemes. 

However your check only covers the `wss` case, and will throw an error if a url uses any of the `http`, `https`, `ws` schemes.